src/components/learning-guide/progress-bar.cjsx

```diff
@@ -6,15 +6,21 @@ ChapterSectionType = require './chapter-section-type'
 module.exports = React.createClass
   displayName: 'LearningGuideProgressBar'
 
+  getDefaultProps: ->
```

Comment:
Because I got paranoid that I missed updating a component and it wouldn't get passed and would break horribly :)